lib/chat/utils/ChatResponseKeys.dart

```diff
@@ -0,0 +1,6 @@
+//TODO not sure about this class, if should be here or if even should exist.
+class ChatResponseKeys {
+  static const keyId = "id";
+  static const keyTitle = "title";
+  static const keyValue = "value";
+}
```

Comment:
As commented right above, not sure about this class, if we should persist that or maybe move it into some other place.

Reply:
we normally would make it private at the point of use, and call it "_Fields"

Reply:
we don't need to repeat the context "key"

Reply:
Ok, changed, moved to private _Fields class at the point of usage, ChatResponseKeys.dart has been removed, **key** annotation at the beginning of each property removed.

Reply:
👍